docs/user-guide/panels.md

```diff
@@ -9,9 +9,9 @@ A case can then be associated with one or several gene panels.
 
 ### File format
 
-The gene panel is a tab separated text file with a header that describes the columns and one line for each gene entry.
+The gene panel is semicolon (;) or tab-separated text file with an optional header that describes the columns and one line for each gene entry.
```

Comment:
```suggestion
The gene panel is a tab, space or semicolon (;)-separated text file with an optional header that describes the columns and one line for each gene entry. You must use the same delimiter for the whole file. Ideally do not use the delimiter characters in other places in the file. Consult with an admin if you need to use the delimiter characters in other fields for help with escaping them or ensuring a higher priority separator is used on a previous line.
```

Reply:
Are you sure that space is allowed? I don't think so. I think I've tried one with space from a customer some time ago and it  didn't work

Reply:
So it should, but it is a little tricky in that if there is a tab somewhere, that will take precedence. More eyerollingly, both have priority over semicolon. See https://github.com/Clinical-Genomics/scout/blob/41b61ba112ed685e39a5029b2d7491643fbe1081/scout/parse/panel.py#L174 and following.

Reply:
If documentation starts getting ridiculous, it is a sign that the corresponding legacy code needs a refactor.

Reply:
> If documentation starts getting ridiculous, it is a sign that the corresponding legacy code needs a refactor.

Perhaps something that checks the file line by line and tells you what is wrong with that line?

Reply:
What about removing the space as a separator directly in the code?

Reply:
That would very likely solve more problems than it would create. If so, check test, fixture and legacy files to be sure.

Reply:
Yeah or just remake/use a regular csv/tsv parser class already that properly handles escapes, delimiters, quotes etc.

> > If documentation starts getting ridiculous, it is a sign that the corresponding legacy code needs a refactor.
> 
> 
> 
> Perhaps something that checks the file line by line and tells you what is wrong with that line?

Reply:
(No offense previous authors - I've done worse hacks, and I'm also a little hungry now.. 😅)

Reply:
Ok, let's revise the parsing, tomorrow!

Reply:
The only place where space as a separator was mentioned and used is in the panel parser, and even there it was not contemplated in the comments, see [L202](https://github.com/Clinical-Genomics/scout/blob/41b61ba112ed685e39a5029b2d7491643fbe1081/scout/parse/panel.py#L202).

I've removed it from the parsing now and it should be fine I think!

Reply:
@dnil I'm going to merge this is you are OK with it!

Reply:
I think this is good, but perhaps ping one or two panel maintainers at CMMS to be sure this doesn't disrupt their workflow? 😊

Reply:
Asked for feedback: CMMS downloads the demo excel file, pasts genes into it and the saves it as a csv, I'll try the procedure with this branch on stage and it it works I'll merge!

Reply:
And of course there are issues, because if you take and excel and you save it as a csv file then the separator is comma! Tried to save it as a tab separated file and the result is better, it just adds annoying quotes around the fields that have comma:

![image](https://user-images.githubusercontent.com/28093618/111302603-e6f2a000-8653-11eb-9746-96ff8b6f3c1c.png)

OMG I hate Excel, I'll fix it so it removes these quotes during the parsing

Reply:
Tested the latest changes:
- [ ] Download example excel file
- [ ] edit
- [ ] Save it as tab separated
- [ ] Import it as a new panel

Works and the quotes are gone: 

![image](https://user-images.githubusercontent.com/28093618/111306014-f70c7e80-8657-11eb-832a-a3cabc679b2b.png)

Reply:
> And of course there are issues, because if you take an excel and you save it as a csv file then the separator is comma! Tried to save it as a tab separated file and the result is better, it just adds annoying quotes around the fields that have comma:
> 
> ![image](https://user-images.githubusercontent.com/28093618/111302603-e6f2a000-8653-11eb-9746-96ff8b6f3c1c.png)
> 
> OMG I hate Excel, I'll fix it so it removes these quotes during the parsing

Agree on that. And comma or semicolon depends on locale. Its not very convenient with comma separation if you have one in every decimal number.. 😆

Reply:
Again, there are loads of csv/tsv parsing libraries. One more dependency, but.. 🤷

Reply:
I suggest let's keep it this way and then if there are still problems we change how we do the parsing. I don't have the energy to completely re-write it now.